library/math/bounded.lisp

```diff
@@ -0,0 +1,57 @@
+;;;; bounded.lisp
+;;;;
+;;;; Numerical types with fixed bounds
+
+(coalton-library/utils:defstdlib-package #:coalton-library/math/bounded
+  (:use
+   #:coalton
+   #:coalton-library/builtin
+   #:coalton-library/classes
+   #:coalton-library/functions)
+  (:export
+   #:Bounded #:minBound #:maxBound))
+
+#+coalton-release
+(cl:declaim #.coalton-impl:*coalton-optimize-library*)
+
+(in-package #:coalton-library/math/bounded)
+
+(coalton-toplevel
+  (define-class (Bounded :a)
```

Comment:
How about IFIX and UFIX? You can use `(lisp Ifix () cl:most-positive-fixnum))` and the like.

Reply:
Also document the class.